extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -1,5 +1,6 @@
 import { SOPClassHandlerName, SOPClassHandlerId } from './id';
 import { utils, classes } from '@ohif/core';
+import metadataProvider from '@ohif/core/src/classes/MetadataProvider';
```

Comment:
You should get the metadataProvider from either @ohif/core, in the line above, or from the init method in the extension.

Reply:
apparently your mention of the `@ohif/ core` has sent notification to all people in the core, this is funny :D

Reply:
now importing metadataProvider from OHIF.classes.MetadataProvider.
Commit 1469ba76b4b2755f50f75b68048798183963a3d8